client-src/elements/chromedash-typeahead.ts

```diff
@@ -123,37 +122,40 @@ export class ChromedashTypeahead extends LitElement {
 
   async handleCandidateSelected(e) {
     const candidateValue = e.detail.item.value;
-    const inputEl = this.slInputRef.value.input;
-    const wholeStr = inputEl.value;
+    const inputEl = this.slInputRef.value?.renderRoot.querySelector('input');
```

Comment:
I think we could avoid repeated use of `!` below by bailing out here if `inputEl`is not found.
`if (!inputEl) return;`

Reply:
sounds good